src/modules/referrals/components/ProjectReferralPostingForm.tsx

```diff
@@ -97,6 +97,10 @@ export const ProjectReferralPostingForm: React.FC<Props> = ({
 
   return (
     <DynamicForm
+      // Use the key to force a re-render when the referral posting changes.
+      // Note: the change to initialValues won't not trigger a form re-render,
```

Comment:
Maybe typo, should it be `change to initialValues won't trigger a form re-render`?

Reply:
whoops thanks